build.zig

```diff
@@ -27,13 +27,15 @@ pub fn build(b: *zbs.Builder) !void {
 
     const examples = b.option(bool, "examples", "Set to true to build examples") orelse false;
 
-    // Sigh, why are the conventions inconsistent like this.
     const resolved_prefix = try std.fs.path.resolve(b.allocator, &[_][]const u8{b.install_prefix.?});
-    if (std.mem.eql(u8, resolved_prefix, "/usr")) {
-        b.installFile("example/init", "../etc/river/init");
-    } else {
-        b.installFile("example/init", "etc/river/init");
-    }
+    const default_config_path = try std.fs.path.resolve(
+        b.allocator,
+        &[_][]const u8{
+            if (std.mem.eql(u8, resolved_prefix, "/usr")) "/" else resolved_prefix,
+            "etc/river/init",
+        },
+    );
+    b.installFile("example/init", default_config_path);
```

Comment:
According to the doc comment on `Builder.installFile()` the second argument should be a path relative to the install prefix. This currently if `Builder.dest_dir` is `null` but will break if `DESTDIR` is set in the environment. This is important for packaging so we should really use a relative path here.

Reply:
https://github.com/ziglang/zig/pull/7646